### ISSUE TITLE
Image optimisation

### ISSUE THREAD

**Issue by jamesgorrie** [wellcometrust/wellcomecollection.org#3991]
Keeping track of things around image optimisation, as it is by far the heaviest part of our site.

---

- [ ] What is our strategy for which images get loaded how? There are currently:
  1. Images that should load on page load at decent quality. e.g. Work main image, article featured image
  1. Images that should be loaded in at poor quality. These are image where having an indication of the image might be useful, and we then load the right width of image in.
  1. Images that should not be loaded in.

Some thoughts on each:

## Decent quality
* Should we be leveraging the fact that we have webp available via the IIIF API?

## Poor quality, then upgrade
* Do we load in the poor quality image only when close to inViewport then upgrade? Seems like it might actually use more bandwidth for nothing. But loading in the poor quality out of bandwidth already uses a lot of bandwidth. See a works search with many results
* Could we be using bitonal images, available via the IIIF API?

## Don't load images till in viewport
* We need to avoid bouncy layouts when loading images in.
* Do we offer a link to the image?
* How do we display the caption, as it feels like it would give good context
* Do we have a universal switch saying, "Actually don't hide my images, I don't use JS, but want images."

## General thoughts
* How do we work out which to serve to who? This is a bigger question I think around cutting the mustard.
